prediction_market_agent/tools/mech/utils.py

```diff
@@ -43,7 +43,27 @@ class MechTool(str, Enum):
     PREDICTION_URL_COT = "prediction-url-cot"
 
 
+def is_package_available(package_name: str) -> bool:
+    """
+    Asserts whether the specified package is available.
+
+    Args:
+        package_name (str): The name of the package to check.
+
+    Returns:
+        True if the package is available, False otherwise.
+    """
+    if find_spec(package_name):
+        return True
+    return False
+
+
 def mech_request(question: str, mech_tool: MechTool) -> MechResponse:
+    if not is_package_available("mech_client"):
+        raise EnvironmentError("Package `mech_client` is not installed.")
+
+    from mech_client.interact import ConfirmationType, interact
+
```

Comment:
Conditionally importing mech_client if it's in the environment, otherwise throws an error.